README.md

```diff
@@ -48,11 +50,11 @@ With this information, a developer can make an informed decision on whether to u
 | Show vulnerabilities inside the project descriptor      |         ✅         |            ✅            |          ✅          |           ✅           |          ❌           |
 | Upgrade vulnerable dependencies to fixed versions       |         ✅         |            ✅            |          ✅          |           ❌           |          ❌           |
 | Automatically trigger a scan upon code changes          |         ✅         |            ❌            |          ✅          |           ❌           |          ❌           |
-| Free project scanning                                   |         ✅         |            ❌            |          ❌          |           ❌           |          ❌           |
+| Unlimited free project scanning                         |         ✅         |            ❌            |          ❌          |           ❌           |          ❌           |
 | Exclude transitive dependencies from project descriptor |         ❌         |            ✅            |          ❌          |           ❌           |          ❌           |
 
 ### Free Go Modules Security Scanning and Metadata from GoCenter
-The JFrog Extension also includes free security scanning and metadata of Go Modules from [JFrog GoCenter](https://search.gocenter.io/). Read more in the [Go Projects](#go-projects) section.
+The JFrog Extension also includes unlimited free security scanning and metadata of Go Modules from [JFrog GoCenter](https://search.gocenter.io/). Read more in the [Go Projects](#go-projects) section.
 
 ## Viewing and Updating Project Dependencies
```

Comment:
The link to this section seems to be broken in the table of contense.
We should change in the table of contense - 
```
- [Viewing Project Dependencies Information](#viewing-project-dependencies-information)
```
-->
```
- [Viewing and Updating Project Dependencies](#viewing-and-updating-project-dependencies)
```